mesheryctl/internal/cli/root/app/import.go

```diff
@@ -59,22 +55,21 @@ Example: mesheryctl app import -f ./application.yml -s "Kubernetes Manifest"`
 		var err error
 		mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
 		if err != nil {
-			utils.Log.Error(err)
-			return nil
+			return utils.ErrLoadConfig(err)
 		}
 
 		appURL := mctlCfg.GetBaseMesheryURL() + "/api/application"
 
 		// If app file is passed via flags
 		if !isValidSource(sourceType) {
-			return errors.Errorf("Application source type (-s) invalid or not passed.\nAllowed source types: %s", strings.Join(validSourceTypes, ", "))
+			return ErrValidSource(validSourceTypes)
 		}
 
 		app, err := importApp(sourceType, file, appURL, true)
 
 		if err != nil {
-			utils.Log.Error(err)
-			return nil
+			utils.Log.Error(ErrImportApp(err))
+			return ErrImportApp(err)
```

Comment:
We don't need to return the meshkit error if we are logging it via meshkit logger

Reply:
@suhail34 to clarify,  so it means throughout all the packages in mesheryctl that has any var that calls a func 
`mctlCfg, err := config.GetMesheryCtl(viper.GetViper())` the error function `ErrInvalidMeshConfig` in the config already returns the error when an error occurs. 
so i would just `utils.Log.Error(err)`

Reply:
with just utils.Log.Error(err) because it is a meshkit error so we do not need to define a meshkit error again